.travis.yml

```diff
@@ -2,6 +2,14 @@
 sudo: required
 dist: trusty
 
+addons:
```

Comment:
can you please remind me why these addons are needed?

Reply:
Uh I wasn't able to build it without it. I can post the error message here by removing them and running tests again if you'd like. I also noticed that it was necessary in the spark-3.0.0 release branch here: https://github.com/combust/mleap/blob/spark-3.0.0/.travis.yml#L14

Reply:
👍